docs/diagrams/AddTaskCommand.puml

```diff
@@ -0,0 +1,104 @@
+@startuml
+!include style.puml
```

Comment:
this is very detailed, but maybe you could magnify it a bit cos have to zoom in to see the details

Reply:
This applies to the rest of the sequence diagrams.